lint/rule_panel_job_instance.go

```diff
@@ -30,7 +30,7 @@ func NewPanelJobInstanceRule() *PanelRuleFunc {
 			}
 
 			for _, target := range p.Targets {
-				node, err := parsePromQL(target)
+				node, err := parsePromQL(target.Expr)
```

Comment:
Is this refactor necessary?  Tends to be bad form to mix refactoring with fixes like this.

Reply:
I did that because in the context of template variable, the query is not part of a target, but only a string.
Now, I can propose to extract the "variables expansion" part of `parsePromQL` in a separate function and use it in both places.
This extraction into another function is anyway something I was planning to submit as I have some clever colleagues who used this trick to have customizable "group by" in their dashboards: `sum by(${variable:csv}) metric{...}` and this makes the linter incorrectly report an error.